sklearn/model_selection/_split.py

```diff
@@ -583,7 +583,7 @@ class StratifiedKFold(_BaseKFold):
         .. versionchanged:: 0.22
             ``n_splits`` default value changed from 3 to 5.
 
-    shuffle : boolean, optional
+    shuffle : bool, default=None
```

Comment:
False

Reply:
Thanks for the feedback @NicolasHug ! I've corrected my work and pushed changes.